docs/trace/customizing-the-sdk/README.md

```diff
@@ -268,15 +268,15 @@ var tracerProvider = Sdk.CreateTracerProviderBuilder()
 
 It is also common for exporters to provide their own extensions to simplify
 registration. The snippet below shows how to add the
-[JaegerExporter](../../../src/OpenTelemetry.Exporter.Jaeger/README.md) to the
+[ZipkinExporter](../../../src/OpenTelemetry.Exporter.Zipkin/README.md) to the
 provider before it is built.
 
  ```csharp
 using OpenTelemetry;
 using OpenTelemetry.Trace;
 
 var tracerProvider = Sdk.CreateTracerProviderBuilder()
-    .AddJaegerExporter()
+    .AddZipkinExporter()
```

Comment:
Same as this https://github.com/open-telemetry/opentelemetry-dotnet/pull/4777/files#r1297351915. We could replace this with OTLP Exporter.

Reply:
Updated it.